StRoot/StFwdTrackMaker/include/Tracker/FwdTracker.h

```diff
@@ -32,10 +32,15 @@
 #include "StFwdTrackMaker/include/Tracker/CriteriaKeeper.h"
 
 #include "GenFit/FitStatus.h"
+#include "GenFit/GFRaveVertexFactory.h"
 
 #include "StFwdTrackMaker/FwdTrackerConfig.h"
 #include "StFwdTrackMaker/Common.h"
 
+namespace FwdTrackerGlobal {
+    bool verbose = true;
+}
+
```

Comment:
Let's not introduce extra global options but rely on the existing framework's functionality

```suggestion
```